logrotate/src/LogRotateHelper.ipp

```diff
@@ -30,6 +30,30 @@
 #include <sstream>
 
 
+#if (defined(WIN32) || defined(_WIN32) || defined(__WIN32__)) && !defined(__MINGW32__)
+
+//http://stackoverflow.com/questions/321849/strptime-equivalent-on-windows
+
+#include <time.h>
+#include <iomanip>
+
+char* strptime(const char* s, const char* f, struct tm* tm) {
```

Comment:
Nice.

You could also have reused the g3log put_time that either use std::put_time or implements it if it's not available